package.json

```diff
@@ -115,19 +115,19 @@
     "ts-node": "10.9.1",
     "tslint": "6.1.3",
     "typescript": "4.9.5",
-    "utf-8-validate": "5.0.10",
+    "utf-8-validate": "6.0.3",
     "uuid": "9.0.0",
-    "ws": "8.11.0",
-    "xml2js": "0.4.23",
+    "ws": "8.13.0",
+    "xml2js": "0.5.0",
     "yaml": "2.2.1",
     "yamljs": "0.3.0",
     "yauzl": "2.10.0",
     "yazl": "2.5.1"
   },
   "devDependencies": {
-    "@types/lodash": "4.14.191",
+    "@types/lodash": "4.14.194",
     "@types/mongodb": "4.0.7",
-    "@types/node": "18.15.3",
+    "@types/node": "18.15.11",
     "@types/pg": "8.6.6",
     "@types/request": "2.48.8",
     "eslint": "8.23.1",
```

Comment:
@liranmauda maybe you can change the `eslint` version to `v8.38.0` as @dannyzaken suggested in #7264?

Reply:
Nope, This one goes into backport, We don't want to carry those